packages/financial-templates-lib/src/price-feed/VaultPriceFeed.js

```diff
@@ -0,0 +1,103 @@
+const { PriceFeedInterface } = require("./PriceFeedInterface");
+const { BlockFinder } = require("./utils");
```

Comment:
Awesome to see this re-use the `BlockFinder` created originally for the BalancerPriceFeed!